GVRf/Framework/jni/engine/importer/assimp_importer.cpp

```diff
@@ -68,4 +71,176 @@ std::shared_ptr<Mesh> AssimpImporter::getMesh(int index) {
 
     return std::shared_ptr < Mesh > (mesh);
 }
+
+void AssimpImporter::sceneRecursion(aiNode* node, const aiScene* aScene, std::shared_ptr<Scene> scnPtr, JNIEnv * env, jobject obj, jobject bitmap, aiMatrix4x4 accumulatedTransform)
+{
+    for(int i=0; i < node->mNumMeshes; i++)
+    {
+        std::shared_ptr<SceneObject> scnObject(new SceneObject());
+
+        // Creates a scene object for the node children meshes.
+        if(scnObject->render_data() == NULL)
```

Comment:
Sorry for this section. It was just a flag to check the render data associated with the scene object. But the part I missed is that this was a check for a newly created scene object which, by default, won't have any render data. So this check won't be of any use as this flag will always be true. I fixed this issue in the latest commit. Thanks for pointing this out.